queries/agda/highlights.scm

```diff
@@ -20,16 +20,17 @@
 ; Set
 (SetN) @type.builtin
 
+(expr . (atom) @function)
+
+((atom) @boolean
+  (#vim-match? @boolean "(true|false|True|False)$"))
```

Comment:
`(#any-of? @boolean "true" "false"  "True" "False")`

Reply:
> `(#any-of? @boolean "true" "false" "True" "False")`

Like this?
![image](https://user-images.githubusercontent.com/60290287/187022577-07654d83-949d-407c-845a-1aeace2c5dc9.png)